Tester.py

```diff
@@ -17,13 +17,21 @@
 
 #analysis
 # message_number_graph = network.message_number_graph(log_data, nicks, nick_same_list)
-nick_change_graph_list =  user.nick_change_graph(log_data)
+#nick_change_graph_list =  user.nick_change_graph(log_data)
 
 #output
 # saver.draw_nx_graph(message_number_graph, output_directory, "message_time_graph")
-for i in range(len(nick_change_graph_list)):
-	saver.draw_nx_graph(nick_change_graph_list[i], output_directory, "nick_change_graph_" + str(i))
+#for i in range(len(nick_change_graph_list)):
+#	saver.draw_nx_graph(nick_change_graph_list[i], output_directory, "nick_change_graph_" + str(i))
 
+# Test for Conversation Length and Conversation Refresh Time
+CL,CRT = channel.conv_len_conv_refr_time(log_data,nicks,nick_same_list)
+saver.save_csv(CL,output_directory,"CL")
+saver.save_csv(CRT,output_directory,"CRT")
+
+# Test for Response Time
+RT = channel.find_response_time(log_data,nicks,nick_same_list)
```

Comment:
This is analysis should be under the analysis comment and output under output comment.